tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js

```diff
@@ -157,6 +157,8 @@ boundaryEventData.forEach(({ type, nodeType, eventXMLSnippet }) => {
 
         waitToRenderAllShapes();
 
+        getElementAtPosition(taskPosition).click();
```

Comment:
Why is this required for the test to pass?

Reply:
The `$el.position()` below is off by about 69 pixels left if I don't do this. Something different about the highlight shape's left position.